docs/reference/transactions.mdx

```diff
@@ -20,42 +19,43 @@ The `proposer` acts similar to the `nonce` in Ethereum transactions, and helps p
 The `payer` is who will be paying for the transaction.
 As mentioned before, it is very important to always provide both a `proposer` and `payer`, the transaction will not work with out them, when in doubt its probably safe to make them the same.
```

Comment:
mutate defaults these values to `authz`, so this no longer applies

Reply:
Changing